app/routes/wallet/credentials.py

```diff
@@ -1,34 +1,64 @@
-from typing import Optional
+from typing import List, Optional
 
 from aries_cloudcontroller import (
     AttributeMimeTypesResult,
-    CredInfoList,
     CredRevokedResult,
-    IndyCredInfo,
-    VCRecord,
-    VCRecordList,
     W3CCredentialsListRequest,
 )
-from fastapi import APIRouter, Depends
+from fastapi import APIRouter, Depends, Query
 
 from app.dependencies.acapy_clients import client_from_auth
 from app.dependencies.auth import AcaPyAuth, acapy_auth_from_header
 from app.exceptions import handle_acapy_call
+from app.models.wallet import CredInfoList, IndyCredInfo, VCRecord, VCRecordList
 from shared.log_config import get_logger
 
 logger = get_logger(__name__)
 
 router = APIRouter(prefix="/v1/wallet/credentials", tags=["wallet"])
 
 
-@router.get("", response_model=CredInfoList)
+@router.get(
+    "",
+    response_model=CredInfoList,
+    summary="Fetch a list of credentials from the wallet",
+)
 async def list_credentials(
     count: Optional[str] = None,
     start: Optional[str] = None,
     wql: Optional[str] = None,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> CredInfoList:
-    """Fetch a list of credentials from the wallet."""
+    """
+    Fetch a list of credentials from the wallet
+    ---
+
+    The WQL (or, wallet query language) parameter can be used to filter credentials returned from the wallet.
+    The `wql` query parameter is a string that can be used to filter records based
+    on the attributes name and value of the record.
+
+    The following string will look for the credential with the attribute `age` with value `21`:
+
+        {"attr::age::value": "21"}
```

Comment:
I think WQL is important enough to show at least a couple different examples of how it can be used.

We show it can fetch credentials by value. It can also fetch credentials with a specific attribute name? Is there also a way of getting credentials where age >= 21? Let's try add examples for those as well if possible

Reply:
Removed link from docstrings as I was unable to reproduce any of their examples. Only the one above.